### PR TITLE
 Add FQDN or URL Custom support for context profile

### PR DESCRIPTION
**Pull request by ksamoray**
To allow addition of custom URLs or FQDNs to context profile objects.

Fixes: #699, #721

Signed-off-by: Kobi Samoray <ksamoray@vmware.com>

### CODE REVIEW

File: nsxt/resource_nsxt_policy_context_profile_custom_fqdn.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	return &schema.Resource{
+		Create: resourceNsxtPolicyContextProfileCustomFQDNCreate,
+		Read:   resourceNsxtPolicyContextProfileCustomFQDNRead,
+		Delete: resourceNsxtPolicyContextProfileCustomFQDNDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"fqdn": {
```
Comment (annakhm) [inline nsxt/resource_nsxt_policy_context_profile_custom_fqdn.go:28]: Should we allow to configure CUSTOM_URL as part of same resource?

Comment (ksamoray) [reply nsxt/resource_nsxt_policy_context_profile_custom_fqdn.go:28]: Yeah we could rename this resource to "custom_attribute" and have both. Would it make this more usable?

File: nsxt/resource_nsxt_policy_context_profile_custom_attribute.go [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+	d.Set("attribute", attribute)
+	return nil
+}
+
+func resourceNsxtPolicyContextProfileCustomAttributeCreate(d *schema.ResourceData, m interface{}) error {
+	var err error
+	key := d.Get("key").(string)
+	attribute := d.Get("attribute").(string)
+	if d.Id() == "" {
+		d.SetId(makeCustomAttributeID(key, attribute))
```
Comment (annakhm) [inline nsxt/resource_nsxt_policy_context_profile_custom_attribute.go:135]: I believe we need to SetId at the end of Create, once we're the resource was created succesfully

Comment (ksamoray) [reply nsxt/resource_nsxt_policy_context_profile_custom_attribute.go:135]: OK

**Review by annakhm — APPROVED** [review]
Lets also add a test that uses custom attribute in context profile. Could be in follow up if you prefer.

**Comment by ksamoray** [comment]
> Lets also add a test that uses custom attribute in context profile. Could be in follow up if you prefer.

Makes sense, I'll create an issue to track this follow up.

**Pull request merged**